src/Frontend/state/helpers/progress-bar-data-helpers.ts

```diff
@@ -212,7 +284,21 @@ export function getUpdatedProgressBarData(args: {
   return progressBarData;
 }
 
-export function getEmptyProgressBarData(): ProgressBarData {
+export function getEmptyProgressBarData(
+  classifications: Classifications,
```

Comment:
should this be nullable?
```suggestion
  classifications?: Classifications,
```

Reply:
No, if there are no classifications that is just an empty object